nixos/modules/services/networking/ntpd.nix

```diff
@@ -34,20 +34,15 @@ in
     services.ntp = {
 
       enable = mkOption {
-        default = !config.boot.isContainer;
+        default = false;
         description = ''
           Whether to synchronise your machine's time using the NTP
           protocol.
         '';
       };
 
       servers = mkOption {
-        default = [
-          "0.nixos.pool.ntp.org"
-          "1.nixos.pool.ntp.org"
-          "2.nixos.pool.ntp.org"
-          "3.nixos.pool.ntp.org"
-        ];
+        default = config.services.timesyncd.servers;
```

Comment:
Might be good to abstract over which service implements the time service by having an option like `networking.timeServers`.